### ISSUE TITLE
Why is not `sigmoid` or `softmax` used in `eval_net` function?

### ISSUE THREAD

**Issue by whenyd** [milesial/Pytorch-UNet#139]
When evaluate the model, the output is just processed simply with [pred = (pred > 0.5).float()](
https://github.com/milesial/Pytorch-UNet/blob/060bdcd69886a3082a6f8fb7746e12d5fca3e360/eval.py#25).
Why not like in the `predict_img` function [as follow](https://github.com/milesial/Pytorch-UNet/blob/060bdcd69886a3082a6f8fb7746e12d5fca3e360/predict.py#L31):
```
# line 31 to 34
if net.n_classes > 1:
    probs = F.softmax(output, dim=1)
else:
    probs = torch.sigmoid(output)
```

**Comment by milesial** [comment]
Yeah, now that the softmax and sigmoid are out of the model, your solution makes more sense for the evaluation.
Please open a pull request so I can merge the changes.

**Comment by whenyd** [comment]
New pr is opened.

**Comment by milesial** [comment]
Closed by #141
